app/models/cart.rb

```diff
@@ -0,0 +1,6 @@
+class Cart < ApplicationRecord
+  belongs_to :user
+  has_many :cart_items, dependent: :destroy
```

Comment:
How does dependent: :destroy work?

Reply:
In this case, when a Cart is destroyed, dependent: :destroy will cause all CartItem records associated with that specific Cart to be automatically destroyed as well.